### ISSUE TITLE
Use emitFileName from config

### ISSUE THREAD

**Issue by ommish** [adelsz/pgtyped#46]
Hi, thank you for this super useful tool!

I just can't get the `emitFileName` in my config working. From what I can tell, it's not actually being used:
https://github.com/adelsz/pgtyped/blob/master/packages/cli/src/index.ts#L146

This is my config- if you see an error on my part I'd appreciate if you could please point it out.
```json{
  "transforms": [
    {
      "mode": "sql",
      "include": "notifications.sql",
      "emitFileName": "../../../types/src/api/sql/notifications.ts"
    }
  ],
  "srcDir": "./src/db",
  "db": {
    "dbName": "api_dev",
    "user": "postgres",
    "password": "postgres",
    "host": "localhost"
  }
}
```

Otherwise happy to put in a PR if you haven't got the time.
Thanks again!

**Comment by adelsz** [comment]
Hi @ommish,
Thanks for reporting this issue!
`emitFileName` had to be disabled back in v0.5. It only allowed specifying one name for all generated files, which was clearly insufficient.

I have just published v0.6.0, replacing `emitFileName` with `emitTemplate`.
This new setting should allow you to configure how generated file names are derived from source file names.
See [this](https://github.com/adelsz/pgtyped/tree/master/packages/cli#config-file) doc for more info on generated file names and the new template format.

**Comment by ommish** [comment]
That's perfect, thank you!!
